### ISSUE TITLE
[Control/UI] unable to interact certain menu by controller in 24.2

### ISSUE THREAD

**Issue by hong620** [long-war-2/lwotc#1275]
not like lastest stable build, current beta and dev build has lot of progress in controller UI
but still some of menu can't interact with controller UI that makes can't progress the gameplay.

each local resistance management menu add construction of relay anttena right after research is finished in early phase of game.
we can see the menu in lwotc but not yet binded gamepad key on this yet, so can't progress the game right now.

not much critical like resistance relay anttenna one but current squad managment menu in armory also not binded key for gamepad yet and also using arrow key to manually interact left side of management UI not working somehow.

not like former builds, Mission Preparing menu seems working fine now.
it doesn't had 'Direct jump to Squad Manegement Menu button' like a control Preparing UI in Mouse, but anyway squad menu itself is currently broken somehow.. so guess not a issue right now.

hope to more players testing on controller for play LWOTC trough Steam Link or Steam PlayTogether kind way.

**Comment by pledbrook** [comment]
Yeah, access to Squad Management has just been disabled for controller users for now.

Build Radio Relay should be accessible if you select a haven and press the Y controller button. Does that not work? That was in beta 3.1 as far as I can tell.
